.circleci/config/commands/go_test.yml

```diff
@@ -116,6 +116,10 @@ steps:
           # reasons unclear.
           export DOCKER_API_VERSION=1.39
 
+          # Hack: Docker permissions appear to have changed; let's explicitly
+          # chmod liberally the docker certificate path.
+          chmod 777 -R $DOCKER_CERT_PATH
```

Comment:
Wouldn't 755 suffice?  Or do we need to write to this dir from within the container?

Reply:
Probably don't actually need to write on this one.